build_functions/CMakeLists.txt

```diff
@@ -0,0 +1,216 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set(trw_internal_dir ${CMAKE_CURRENT_LIST_DIR} CACHE INTERNAL "")
```

Comment:
>trw [](start = 4, length = 3)

trw?

Reply:
Means trsw - test runner switcher

---
In reply to: [608140769](https://github.com/Azure/c-testrunnerswitcher/pull/65#discussion_r608140769) [](ancestors = 608140769)